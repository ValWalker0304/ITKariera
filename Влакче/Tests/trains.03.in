Add 58651 Shumen P 3
Add 73224 Burgas P 4
Add 21407 Ruse P 10
Add 50000 Burgas P 1
Add 28151 Shumen P 3
Add 33991 Karlovo P 3
Travel
Travel
Add 82418 Sofia P 10
Add 82139 Ruse P 8
Add 60619 Plovdiv P 6
Travel
Travel
Add 55384 Mezdra P 8
Travel
Travel
Add 74677 Sindel P 6
Travel
Travel
Travel
Travel
End