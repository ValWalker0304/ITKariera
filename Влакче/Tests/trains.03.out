33991 Karlovo P 3
28151 Shumen P 3
60619 Plovdiv P 6
82139 Ruse P 8
55384 Mezdra P 8
82418 Sofia P 10
74677 Sindel P 6
50000 Burgas P 1
21407 Ruse P 10
73224 Burgas P 4
